packages/eslint-plugin-office-addins/src/rules/call-sync-after-load.ts

```diff
@@ -13,7 +13,7 @@ export = {
     type: <"problem" | "suggestion" | "layout">"suggestion",
     messages: {
       callSyncAfterLoad:
-        "Call context.sync() after calling load on '{{name}}' for property '{{loadValue}}' and before reading the property.",
+        "Call context.sync() after calling load on '{{name}}' for '{{loadValue}}' and before reading the property.",
```

Comment:
Ricky said use: 

"Call context.sync() after calling load on range for the property ‘address’ and before reading the property."

which translates to:

"Call context.sync() after calling load on '{{name}}' for the property '{{loadValue}}’ and before reading the property."

Reply:
Sure, thanks for seeing that.